Casks/n/nextcloud.rb

```diff
@@ -14,13 +14,17 @@
     livecheck do
```

Comment:
```suggestion
    # Upstream publishes releases for multiple different minor versions and the
    # "latest" release is sometimes a lower version. Until the "latest" release
    # is reliably the highest version, we have to check multiple releases.
    livecheck do
```

The `GithubReleases` strategy is only used when `GithubLatest` isn't sufficient, so we should document why it's necessary. Sometimes we have to use `GithubReleases` due to a situation like this and can eventually switch back to `GithubLatest` if/when the upstream situation changes, so this helps us to determine if it's still necessary when reviewing a package.